conda/recipes/numba-cuda/meta.yaml

```diff
@@ -28,6 +28,7 @@ requirements:
   run:
     - python
     - numba >=0.59.1
+    - cuda-core
```

Comment:
Should we either make this optional (like wheels) or should we also add `cuda-python` (as wheels do with their CUDA 11/12)?